imports/components/cards/cards.Activity.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import moment from "moment";
+import { Link } from "react-router";
+import { Error, Success } from "../icons";
+
+const getStyles = (status: string) => {
```

Comment:
@richarddubay this is looking great! The only other thing we should change is the background colors!

`background--dark-primary`
`background--alert`
`background--primary`